Travel Scheduler/Views/PlacesToVisitTableViewCell.m

```diff
@@ -7,40 +7,41 @@
 //
 
 #import "PlacesToVisitTableViewCell.h"
+#import "PlacesToVisitCollectionView.h"
+#import "AttractionCollectionCell.h"
+#import "DetailsViewController.h"
 
-@implementation PlacesToVisitCollectionView
+@interface PlacesToVisitTableViewCell () <UICollectionViewDataSource, UICollectionViewDelegate, AttractionCollectionCellDelegate>
 @end
 
 @implementation PlacesToVisitTableViewCell
 
-#pragma mark - Selecting cell animation
-
-- (void)setSelected:(BOOL)selected animated:(BOOL)animated{
-    [super setSelected:selected animated:animated];
+#pragma mark - Initiation Methods
+- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
+{
+    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
+    [self initAllProperties];
```

Comment:
Maybe createAllProperties to avoid using init?

Reply:
Done!